source/utilities.tex

```diff
@@ -504,13 +504,13 @@
 \end{codeblock}
 
 \pnum
-Constructors and member functions of \tcode{pair} shall not throw exceptions unless one of
+Constructors and member functions of \tcode{pair} do not throw exceptions unless one of
 the element-wise operations specified to be called for that operation
 throws an exception.
 
 \pnum
-The defaulted move and copy constructor, respectively, of \tcode{pair} shall
-be a constexpr function if and only if all required element-wise
+The defaulted move and copy constructor, respectively, of \tcode{pair}
+is a constexpr function if and only if all required element-wise
```

Comment:
Right now the grammar is: "The X and Y, respectively, is Z if A and B, respectively, satisfies some condition."
I guess you're suggesting: "The X and Y, respectively, are Z if A and B, respectively, satisfy some condition."
Another alternative: "The X or Y, respectively, is Z if A or B, respectively, satisfies some condition."

FWIW, I think this grammar is just too far from proper to have a "right" answer, but my guess was that the singular is more correct — because it's possible for X to be Z independently of Y. Each of _X and Y is Z_ independently of the other.

Reply:
> FWIW, I think this grammar is just too far from proper to have a "right" answer

Amen to that: this sentence is a painfully awkward demonstration of the lengths we'll go to in order to avoid repetition. We should probably simply strike it, since
* "shall be a constexpr function" doesn't mean what the Library thinks it means,
* the core language tells us when defaulted constructors are `constexpr`, there's no need to repeat it here, and
* [functions.within.classes] says we "do not describe copy/move constructors, assignment operators, or (non-virtual) destructors with the same apparent semantics as those that can be generated by default".

but that may be a bit of a stretch for an editorial change.

Feel free to ignore this comment, accept it, and/or file an LWG issue to strike the paragraph altogether.

Reply:
This is awful. But it's pre-existing, so I'm not blocking this pull request on it. If someone wants to take some additional action here, please do! (I think I'll change the "and"s to something else as a short-term "fix".)

Reply:
Also, hold on just a minute, this is saying that the defaulted move constructor is constexpr if all copies are valid and the defaulted copy constructor is constexpr if all moves are valid. What the ...?

Reply:
> If someone wants to take some additional action here

Ideally, additional action involving cleansing flame.